src/api/useFhirDefinitionsService.ts

```diff
@@ -48,24 +55,50 @@ export class FhirDefinitionsServiceApi {
     }
   }
 
-  async getResourceTree(resourceName) {
+  async getResourceTree(resourceId): Promise<StructureDefinitionDto> {
     try {
       const response = await axios.get<any>(
-        `${this.baseUrl}/structure-definitions/${resourceName}`
+        `${this.baseUrl}/qicore/resources/structure-definitions/${resourceId}`,
+        {
+          headers: {
+            Authorization: `Bearer ${this.getAccessToken()}`,
+          },
+        }
       );
       return response.data;
     } catch (error) {
       console.error(
-        `An error occurred while loading definition for resourceName [${resourceName}]: `,
+        `An error occurred while loading definition for resourceId [${resourceId}]: `,
+        error
+      );
+    }
+    return null;
+  }
+
+  async getFhirValueSetExpansion(valueSetId: string): Promise<any> {
+    try {
+      const response = await axios.get<any>(
+        `https://tx.fhir.org/r4/ValueSet/${valueSetId}/$expand`,
+        {
+          headers: {
+            Accept: "application/fhir+json",
+          },
+        }
+      );
+      return response.data;
+    } catch (error) {
+      console.error(
+        `An error occurred while loading definition for resourceName [${valueSetId}]: `,
         error
       );
     }
     return null;
   }
 
   getBasePath(resource: any): string {
-    const elements = [...resource?.definition?.snapshot?.element];
-    return elements?.[0].path;
+    // const elements = [...resource?.definition?.snapshot?.element];
+    // return elements?.[0].path;
```

Comment:
Debugging code?

Reply:
likely....will remove in an upcoming commit